src/grib2.h

```diff
@@ -150,15 +150,11 @@
 #define _grib2_H
 #include<stdio.h>
 
-#define G2_VERSION "g2clib-1.6.0"
+#define G2_VERSION "g2clib-1.6.3"
 
-#ifdef __64BIT__
-typedef int g2int;
-typedef unsigned int g2intu;
-#else
 typedef long g2int;
 typedef unsigned long g2intu;
-#endif
+
```

Comment:
If you removed `__64BIT__`, you should define `g2int` and `g2intu` in terms of fixed width integer types. Again, please see:

https://github.com/NOAA-EMC/NCEPLIBS-g2c/commit/3f7ed54a03415c66e20dd86c79b7b6073b4ae2fa

In fact this PR should not be combined with the changes made in:
 
https://github.com/NOAA-EMC/NCEPLIBS-g2c/pull/28